TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -62,6 +65,16 @@ namespace Plugin {
         JsonObject config;
         config.FromString(service->ConfigLine());
 
+        InputValidation::Instance().addValidator("ttsendpoint", ExpectedValues<std::string>("^https?://[a-zA-Z0-9]+.*"));
+        InputValidation::Instance().addValidator("ttsendpointsecured", ExpectedValues<std::string>("^https://[a-zA-Z0-9]+.*"));
+        InputValidation::Instance().addValidator("speechid", ExpectedValues<uint32_t>("^[0-9]+$"));
+        InputValidation::Instance().addValidator("speechid", (std::function<bool(const uint32_t &)>)[](const uint32_t &val) -> bool { return val > 0; });
+        InputValidation::Instance().addValidator("text", ExpectedValues<std::string>("^[a-zA-Z0-9]+"));
+        InputValidation::Instance().addValidator("speechrate", ExpectedValues<std::string>({"slow", "medium", "fast", "faster", "fastest"}));
+        InputValidation::Instance().addValidator("rate", (std::function<bool(const uint8_t &)>)[](const uint8_t &val) -> bool { return val==0 || (val >=1 && val <=100); });
```

Comment:
val >= 0 && value <= 100? BTW, in which scenario value 0 is accepted?

Reply:
The cases, where the label "rate" is not present the rate value is set to 0. Which means that, in case setconfiguration is called other than rate parameter input validator should return true.

Reply:
Ok. In that case, can you configure the ExpectedValues<uint8_t>(0, 100)?